packages/lesspass-pure/src/components/MasterPassword.vue

```diff
@@ -33,26 +33,44 @@
         v-on:input="updateValue($event.target.value)"
         v-on:keyup.enter="$emit('keyupEnter')"
       />
-      <span class="input-group-btn" v-if="fingerprint && value" v-on:click="togglePasswordType">
+      <span
+        class="input-group-btn"
+        v-if="fingerprint && value"
+        v-on:click="togglePasswordType"
+      >
         <button id="fingerprint" class="btn" type="button" tabindex="-1">
           <small>
-            <i class="fa fa-fw" v-bind:class="[icon1]" v-bind:style="{ color: color1 }"></i>
-            <i class="fa fa-fw" v-bind:class="[icon2]" v-bind:style="{ color: color2 }"></i>
-            <i class="fa fa-fw" v-bind:class="[icon3]" v-bind:style="{ color: color3 }"></i>
+            <i
+              class="fa fa-fw"
+              v-bind:class="[icon1]"
+              v-bind:style="{ color: color1 }"
+            ></i>
+            <i
+              class="fa fa-fw"
+              v-bind:class="[icon2]"
+              v-bind:style="{ color: color2 }"
+            ></i>
+            <i
+              class="fa fa-fw"
+              v-bind:class="[icon3]"
+              v-bind:style="{ color: color3 }"
+            ></i>
           </small>
         </button>
       </span>
     </div>
-    <button
-      id="encryptMasterPassword__btn"
-      type="button"
-      class="btn btn-link btn-sm p-0"
-      v-if="showEncryptButton"
-      v-on:click="encryptMasterPassword()"
-      v-bind:class="{'disabled': email === ''}"
-    >
-      <small>{{ EncryptButtonText }}</small>
-    </button>
+    <small>
+      <div class="form-check form-switch" v-if="showEncryptButton">
```

Comment:
Just for the record, this widget will be a [switch in Bootstrap 5](https://getbootstrap.com/docs/5.0/forms/checks-radios/#switches), but for now there's no switch widget in Bootstrap 4 :cry:

Reply:
Checkbox is consistent with the main view. I will see if we can do the bootstrap 5 migration easily without breaking everything.